src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
```

Comment:
```suggestion
  const cost = 40 * days;
```
and use without duplication